README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Slack](https://img.shields.io/badge/slack-@cncf/otel/dotnet-brightgreen.svg?logo=slack)](https://cloud-native.slack.com/archives/C01N3BC2W7Q)
 [![Build](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/actions/workflows/ci.yml)
+[![codecov.io](https://codecov.io/gh/open-telemetry/opentelemetry-dotnet-contrib/branch/main/graphs/badge.svg?)](https://codecov.io/gh/open-telemetry/opentelemetry-dotnet-contrib/)
```

Comment:
Do we want this at all? If the answer is yes, I think we should have some rules established.

1. each component should have a minimum code coverage bar, if any component doesn't meet the bar for a certain period of time, it'll be kicked out from the repo
2. the code coverage infrastructure should be stable (e.g. different runs should give the same/stable results)

An alternative way, is to have code coverage stats on individual components README file, and the component owner can decide what's their bar.

Reply:
I would prefer to have it on an individual component level than at a repo level.